shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -491,6 +491,7 @@ - (BOOL)runWithEntrypoint:(NSString*)entrypoint {
 
   FlutterProjectArgs flutterArguments = {};
   flutterArguments.struct_size = sizeof(FlutterProjectArgs);
+  flutterArguments.enable_impeller = _project.enableImpeller;
```

Comment:
Should we set `enable_impeller` to `false` explicitly on Windows and Linux?

Reply:
I don't think it matters. We'll keep the default false until we have all backends wired up and tested